demo-cat-gallery/src/main/kotlin/com/google/android/flexbox/apps/catgallery/CatViewHolder.kt

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright 2017 Google Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.android.flexbox.apps.catgallery
+
+import android.support.annotation.DrawableRes
+import android.support.v7.widget.RecyclerView
+import android.view.View
+import android.widget.ImageView
+import com.google.android.flexbox.FlexboxLayoutManager
+
+/**
+ * ViewHolder that represents a cat image.
+ */
+internal class CatViewHolder(itemView: View) : RecyclerView.ViewHolder(itemView) {
+
+    private val mImageView = itemView.findViewById(R.id.imageview) as ImageView
```

Comment:
Please rename this to imageView to follow the Kotlin coding conventions https://github.com/JetBrains/kotlin-web-site/blob/64218469db3f746ab1f4a1c4c569c21d654d665f/pages/docs/reference/coding-conventions.md